### DISCUSSION TITLE
macs2 default parameters

### DISCUSSION THREAD

**Discussion by cv55** [stuart-lab/signac#1169] [Q&A] [answered]
Hello, thanks for the great tool,

i was wondering why the peak calling implementation of macs2 in Signac by default uses the "BED" file format, and not the "BEDPE", since scATAC has paired end reads. 
Is it only because the BED format also allows the user to change the central point of the fragment (shift -100)?

Thanks!

**Comment by timoast** [comment, marked as answer]
See https://github.com/timoast/signac/issues/682
